### ISSUE TITLE
locale option not working

### ISSUE THREAD

**Issue by hans0126** [jkuri/ng2-datepicker#176]
`
this.options = new DatePickerOptions();
this.options.locale = "ja";
`
is not working

**Comment by elliotwesoff** [comment]
Also not working for me when I use 'de.'  Bump.

**Comment by jkuri** [comment]
yes, you need to import locales from `moment`, something like this in your component;
```ts
import 'moment/locale/fr';
moment.locale('fr');
```

not sure, but maybe this has to be added to webpack config:
```js
`new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /en/)`
```

if nothing of this works, please let me know and I will switch to `date-fns` as `moment` is too large for that simple component anyway. thanks.

**Comment by Simon-edu-Sharing** [comment]
For me,

```
import * as moment from 'moment';
moment.locale('de');
```

worked just fine. Thanks!

**Comment by bchukiat** [comment]
I face the same issue and my application require to change locale dynamically, therefore I modify the **ng2-datepicker.component.ts** by add below line on top of ngOnInit() as below


 ...
 ...
ngOnInit() {

    if(this.options.locale) {
      moment.locale(this.options.locale)
    }
    else {
      moment.locale('th'); // default locale
    }
 ...
 ...

Then add component to our app template file (app.component.html) as example (https://github.com/jkuri/ng2-datepicker/blob/master/src/app/app.component.html)


<ng2-datepicker [options]="options" [(ngModel)]="date"  ></ng2-datepicker>


And set option in app.component.ts as below

 ...
 ...
 ngOnInit() {

      this.options = new DatePickerOptions();
      this.options.locale = 'th';
      this.options.format = "l";
  }
 ...
 ...

**Comment by MattiaMalandrone** [comment]
works fine with months, but days of the week still in english :(
